<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>billet</class>
 <widget class="QDialog" name="billet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>455</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI Semibold</family>
    <pointsize>10</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable Display Semib</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ajout Billet :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable Display Semib</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Séance :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Prix :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Date d'achat :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom Client :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Nom_Client">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>301</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Prix">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>301</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_Achat">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>301</width>
     <height>25</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2023</year>
     <month>12</month>
     <day>11</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>301</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Insert_Data">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert Data</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Prenom_Client">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>301</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Prénom Client :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Filme :</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_Filme">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>297</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre De Places Restant :</string>
   </property>
  </widget>
  <widget class="QLabel" name="nbrPlace">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>321</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Affichage">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>400</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>affichage</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
